app/components/language-menu.tsx

```diff
@@ -175,4 +175,10 @@ const icons: Record<(typeof languageValues)[0], JSX.Element> = {
+        <path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 25.355469 19.753906 C 25.355469 19.75 25.355469 19.75 25.355469 19.75 C 25.390625 19.628906 25.425781 19.511719 25.457031 19.386719 C 26.785156 14.09375 23.542969 7.839844 18.058594 4.546875 C 20.460938 7.804688 21.523438 11.75 20.582031 15.203125 C 20.496094 15.511719 20.394531 15.804688 20.285156 16.09375 C 20.160156 16.011719 20.007812 15.921875 19.804688 15.808594 C 19.804688 15.808594 14.347656 12.441406 8.433594 6.480469 C 8.277344 6.324219 11.585938 11.210938 15.339844 15.175781 C 13.574219 14.183594 8.640625 10.597656 5.519531 7.742188 C 5.90625 8.378906 6.359375 8.996094 6.863281 9.589844 C 9.46875 12.894531 12.867188 16.972656 16.941406 20.105469 C 14.078125 21.855469 10.035156 21.992188 6.011719 20.105469 C 5.015625 19.640625 4.078125 19.078125 3.210938 18.4375 C 4.917969 21.164062 7.539062 23.515625 10.734375 24.886719 C 14.542969 26.523438 18.332031 26.414062 21.152344 24.914062 L 21.148438 24.917969 C 21.164062 24.910156 21.179688 24.898438 21.191406 24.890625 C 21.308594 24.828125 21.421875 24.765625 21.535156 24.699219 C 22.890625 23.996094 25.570312 23.28125 27.007812 26.078125 C 27.359375 26.761719 28.105469 23.136719 25.355469 19.753906 Z M 25.355469 19.753906 "/>
+    </svg>
+  ),
```

Comment:
This icon seems to be causing rendering error, I suspect because of the `style` attribute. 

I would like to try to use a consistent icon style for languages, can we use https://github.com/jesseweed/seti-ui/blob/master/icons/swift.svg?   

For some reason this one is black but it should be possible to change this to the actual swift orange like in VS Code: 

<img width="25" height="24" alt="Image" src="https://github.com/user-attachments/assets/b55f983a-aaaf-491d-8cd9-ea63ffe73124" />

Reply:
Strange, I was testing it in here okay - https://codebeautify.org/svg-viewer#google_vignette - I took it from the official apple site